src/main/java/org/terasology/spawning/SpawnerSystem.java

```diff
@@ -262,9 +272,8 @@ public void update(float delta) {
                 }
 */
                 //TODO check for bigger creatures and creatures with special needs like biome
-
                 // In case we're doing ranged spawning we might be changing the exact spot to spawn at (otherwise they're the same)
-                Vector3f spawnPos = originPos;
```

Comment:
The Oreons should be spawned from the Portals itself, so `spawnPos` must be same as the `originPos`, so as to make the Oreons appear from the Portals itself.